src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -109,12 +122,11 @@ value="$services.localization.render('admin.save')" /></span>
 </div>
 </form>
 
-
 <div id="driveOnNowWhat">
 
-#tp("nowWhat1", ['[[','>>GoogleApps.TestDocumentList]]'])
+    <p>#tp("nowWhat1", ['[[',">>${xwiki.getDocument('GoogleApps.TestDocumentList').getURL()}]]"])</p>
```

Comment:
You can also use the shorter version to get an URL of an XWiki page: `$xwiki.getURL('GoogleApps.TestDocumentList')`

Reply:
thanks.
paul